City-Bureau/city-scrapers-indianapolis
# 🏗️ Build spider: Indianapolis International Airport Board

## What's this PR do?
Adds a spider to scrape meetings from the website of Indianapolis International Airport Board. The new spider is called `ind_iia`.

## Why are we doing this?
Requested by our site partners.

## Steps to manually test

1. Ensure the project is installed:
```
pipenv sync --dev
```
2. Activate the virtual env and enter the pipenv shell:
```
pipenv shell
```
3. Run the spider:
```
scrapy crawl ind_iia -O test_output.csv
```
4. Monitor the output and ensure no errors are raised.

5. Inspect `test_output.csv` to ensure the data looks valid. Target website is [here]().

## Are there any smells or added technical debt to note?
